cdi/embedded/src/test/resources/infinispan.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
```

Comment:
Couldn't we remove the `xsi:schemaLocation` and configure the IDE to use the local XSD for "urn:infinispan:config:9.1"? Before Alpha1 there isn't going to be a 9.2 schema on `www.infinispan.org` anyway.